packages/server/src/api/controllers/component.ts

```diff
@@ -20,7 +20,7 @@ export async function fetchAppComponentDefinitions(ctx: UserCtx) {
     const definitions: { [key: string]: any } = {}
     for (let { manifest, library } of componentManifests) {
       for (let key of Object.keys(manifest)) {
-        if (key === "features") {
+        if (key === "features" || key === "typeSupportPresets") {
```

Comment:
What do these two manifest keys do exactly? would be good to have a comment, or say an array to check against like `MANIFEST_KEYS_THAT_DO_SOMETHING_SPECIAL.includes(key)`

Reply:
They're keys used by the frontend that provide access to features based on the client version. They're specified here to avoid having the `@budibase/whatever` prefix appended to them like the components in the manifest. 

Good idea to note down via code or a comment what it does, will do.